x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/repositories/ClearCcrRestoreSessionRequest.java

```diff
@@ -40,14 +51,35 @@ public ActionRequestValidationException validate() {
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         out.writeString(sessionUUID);
+        if (out.getTransportVersion().onOrAfter(TransportVersion.V_8_8_0)) {
+            shardId.writeTo(out);
+        }
```

Comment:
Since RCS 2.0 is after 8.8, dropping this field for older versions is not an issue at all.